source/tests/infer/gen_dpa1.py

```
@@ -0,0 +1,209 @@
+    e1, f1, v1, ae1, av1 = dp.eval(coord, box, atype, atomic=True)
+    print(f"\n// PBC total energy: {e1[0, 0]:.18e}")  # noqa: T201
+    print_cpp_values("PBC reference values", ae1, f1, av1)
+
+    # ---- 5. Run inference for NoPbc test ----
+    e_np, f_np, v_np, ae_np, av_np = dp.eval(coord, None, atype, atomic=True)
+    print(f"\n// NoPbc total energy: {e_np[0, 0]:.18e}")  # noqa: T201
+    print_cpp_values("NoPbc reference values", ae_np, f_np, av_np)
+
+    # ---- 6. Verify .pth gives same results ----
+    dp_pth = DeepPot(pth_path)
+    e_pth, f_pth, v_pth, ae_pth, av_pth = dp_pth.eval(coord, box, atype, atomic=True)
```

Ruff (0.15.4)

[warning] Unpacked variable `v_pth` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)